### ISSUE TITLE
Update Collection does not return Object

### ISSUE THREAD

**Issue by hugsbrugs** [deployd/deployd#492]
Hi,

In one of my modules, I'm updating a collection using http://docs.deployd.com/docs/developing-modules/internal-api/store.html#s-Store.update%28query,%20changes,%20fn%29-1117

```javascript
var WebPages = process.server.createStore('webpages');
WebPages.update({"id":pageId}, webpage, function(err, webpageResult)
{
    if(err)
    {
        // An error occured adding a webpage
        reject(Error("An error occured getting a webpage : " + err));
    }
    else
    {
        console.log("webpageResult : " + webpageResult);
        resolve(webpageResult);
    }
});     
```

Everything runs fine except in the response callback. My webpageResult is always **undefined** but I'm expecting either a boolean or the updated object. Docs are not very clear about it. How do you check query has been successfull because if I give a non existing id for the query, I won't have any error and I'm unable to check it (or I should first do a request to test the id ...)

Thanks for highlighting me !

**Comment by andreialecu** [comment]
I fixed this in PR #501 , it will return an object with the number of records affected, so that  `webpageResult.n` will be 1 in your code above

**Comment by hugsbrugs** [comment]
very nice : thanks @andreialecu
